portfolio/src/main/webapp/script.js

```diff
@@ -80,8 +81,8 @@ function validateForm() {
 }
 
 function loadComments() {
-    const numValue = document.getElementById('numComments').value;
-    fetch('/data?numValue=' + numValue).then(response => response.json()).then((Data) => {
+        const numValue = document.getElementById('numComments').value;
```

Comment:
fix indentation?

Reply:
At what point or line in the code? I used 3 different JavaScript beautifiers. They all returned this same indentation.